include/dbgroup/atomic/mwcas/lock_free/mwcas_descriptor.hpp

```diff
@@ -241,27 +289,26 @@ class alignas(kCacheLineSize) MwCASDescriptor
    * @param desired A desired value to be embedded.
    * @return The number of followers.
    */
-  auto Finalize(            //
+  void Finalize(            //
       uint64_t desc_addr,   //
       MwCASTarget &target,  //
-      uint64_t desired)     //
-      -> uint32_t;
+      uint64_t desired);
 
   /*############################################################################
    * Internal member variables
    *##########################################################################*/
 
-  /// @brief The status of this CASN descriptor.
+  /// @brief The status of this LFMwCAS descriptor.
```

Comment:
これコピペのときに面倒なので、固有名詞は除いていいかもですね。


```suggestion
  /// @brief The status of this descriptor.
```